projects/visual_bert/configs/localized_narratives/coco2017/masked.yaml

```diff
@@ -0,0 +1,37 @@
+dataset_config:
```

Comment:
This looks like the base dataset config `mmf/configs/datasets/coco2017/masked.yaml`. I don't think we need to add this here again.